PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+# Pull Request Template
+
+## Description
+
+Please include a summary of the work done and which issue is addressed.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix
+- [ ] New feature (new functionality)
+- [ ] This change requires a documentation update(like wiki/requirements or design update)
```

Comment:
I think this line is great to keep track of updates we have to do in documentation, but if there is a new functionality doesn't it automatically mean that it requires a documentation update? So is this line really necessary?

Reply:
Does a change always result in a doc update?

I don't think this is the case, unless it causes a change in the requirements or design files(class diagram/sequence diagram). For example, if someone now opens a PR implementing "Registration", we are not going to need a doc update, since it's all already there. But if someone implements a visualization tool, we'll need an update. 

I wanted to differentiate these cases, giving rise to better communication between the author and whoever will update the doc, reminding the author that she needs to open an issue.

Am I missing a point?

Reply:
A PR does not have to result in a documentation update. I suppose there is going to be PRs for small bug fixes or refactoring in the future. And they probably won't need a doc update. Or some PRs will be the implementation of an already documented feature etc. I think the line can stay.